.gitignore

```diff
@@ -0,0 +1,4 @@
+### Helm ###
+# Chart dependencies
+**/charts/*.tgz
+artifacts/** */
```

Comment:
What does the space */ means here?

Reply:
fixed to `artifacts/` which covers what we need